### ISSUE TITLE
Add CI

### ISSUE THREAD

**Issue by rth** [grimmer0125/webpack-pyodide-demo#3]
It would be useful to add some CI, for instance running on the dev version of pyodide.

**Comment by grimmerk** [comment]
It is great to have one. I prefer to use GitHub action than CircleCI a little more since GitHub one is the built-in feature, but the latter is OK. At this moment, the CI may be the same as https://github.com/pyodide/pyodide/pull/1900 if it targets the dev of Pyodide. For the short-term and long-term differences/goals, what do you think?

**Comment by rth** [comment]
Yes, sure Github Actions would work. In the main repo we use CircleCI mostly for historical reasons and because it has features GH Actions still doesn't have for complicated build pipelines.

**Comment by grimmerk** [comment]
I can do it after this weekend's conference.
